ocs_ci/ocs/ui/pvc_ui.py

```diff
@@ -80,15 +77,13 @@ def verify_pvc_ui(
     ):
         """
         Verifying PVC details via UI
-
         Args:
```

Comment:
Why did you delete the  space line in this file?

Reply:
removed, added by mistake I think